pettingzoo/utils/wrappers/multi_episode_env.py

```diff
@@ -59,6 +70,18 @@ def step(self, action: ActionType) -> None:
             None:
         """
         super().step(action)
+
+        # adjust utilities if this param is enabled
+        if self._starting_utility:
+            for agent in self.agents:
+                self._agent_utilities[agent] = (
+                    self._agent_utilities[agent] - self.rewards[agent]
```

Comment:
This looks like it always subtracts the reward. Shouldn't it just always add it, that way it's like total rewards and will add if you get positive reward or subtract if it's negative?